src/term/transform/resolve_refs.rs

```diff
@@ -124,7 +124,6 @@ impl Term {
         }
         Term::Lst { .. } => unreachable!("Should have been desugared already"),
         Term::Lnk { .. } | Term::Ref { .. } | Term::Num { .. } | Term::Str { .. } | Term::Era | Term::Err => {
-          ()
```

Comment:
You should run cargo fmt and cargo clippy

Reply:
I already did it

Reply:
Should cargo format be changing this into
```
erm::Lnk { .. } | Term::Ref { .. } | Term::Num { .. } | Term::Str { .. } | Term::Era | Term::Err => {}
```
Without the line break?

Reply:
I think it should, but it does not format this way because we are using `max_width = 110`